libcaf_test/caf/test/reporter.cpp

```diff
@@ -100,7 +101,10 @@ struct colorizing_iterator {
         if (c == ')') {
           out->flush();
           *out << term::reset;
-          mode = normal;
+          if (start == off)
+            mode = off;
+          else
+            mode = normal;
```

Comment:
I believe you have tried to fix a case where the uncolored mode jumps back to colored here. But the issue is actually in `case off_escape`. It should not go to state `color` but to `off_color` instead.

Basically, the `off_` states mirror the regular states and should only transition to other `off_` states (or to `verbatim`).

```suggestion
          mode = normal;
```